syft_tensorflow/attributes/attributes.py

```diff
@@ -107,9 +106,16 @@ def __init__(self, tensorflow: ModuleType, hook: "TensorFlowHook"):
 
         self.command_guard = self._command_guard
 
-        # Dict {method_name: <is_inplace:bool>
+        self.exclude = [
+            '__getattr__',
```

Comment:
Why are we excluding `__getattr__`? Also the attributes below should not be here unless totally necessary.